src/js/languages/default.json

```diff
@@ -8,6 +8,11 @@
     "dropPrompt": "Drop file here or",
     "dropPromptMultiple": "Drop files here or",
     "files": "files",
+    "maxSizeSingle": "File exceeds size limit. Select a file with max size of {maxSize}.",
+    "maxSizeMultiple": {
+      "singular": "1 File exceeds size limit. Select files with a max size of {maxSize}.",
+      "plural": "{numOfFiles} Files exceed size limit. Select files with a max size of {maxSize}."
```

Comment:
I used `largeFiles` above but if you stick with this, drop the Of (ie. `numFiles`).
Also these are very computer-ish sounding and *max* isn't a word.  How about something more like:
```
"maxSizeSingle": "The file is too large. Select a file no larger than {maxSize} bytes.",
"maxSizeMultiple": {
   "singular": "One file is too large. Select files which are no larger than {maxSize} bytes.",
    "plural": "{numOfFiles} files are too large. Select files which are no larger than {maxSize} bytes.`
},
```
The units and value for maxSize may need to be more friendly depending on if they can specify units with the value.

Reply:
I definitely liked the naming of `largeFiles` instead so this will be getting implemented with the next push. Thanks for the help on this!